src/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  font-size: 100px;
```

Comment:
better to set font styles in html

Reply:
ready, add it to " * "

Reply:
- comment was to font-family and font-size
- HTML does not mean *

Reply:
> * comment was to font-family and font-size
> * HTML does not mean *

ready